### PR TITLE
Portfolio - DOM

### PR DESCRIPTION
**Pull request by danifromecuador**
In this pull request, I'm creating the mobile button using javascript"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
         <div class="my-logo">
           <a href="https://github.com/danifromecuador/portfolio" class="my-portfolio">My Portfolio</a>
         </div>
-        <div class="menu">
+        <div class="hamburger-icon">
```
Comment (anagudelogu) [inline index.html:23]: - [ ] You did a great job working on this milestone! 🥇 However, please make sure that the hamburger icon only appears in the mobile version.  🤓 🛠️ 

|Template|Your design|
|---|---|
| image | image |

File: script.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+hamburgerIcon.addEventListener('click', () => {
+  hamburgerMenu.style.display = 'block';
+});
+closeIcon.addEventListener('click', () => {
+  hamburgerMenu.style.display = 'none';
+});
+pacMobile.addEventListener('click', () => {
+  hamburgerMenu.style.display = 'none';
+});
```
Comment (anagudelogu) [inline script.js]: - [x] **[OPTIONAL]:** You did a great job writing JavaScript! 💯 However, at the moment, users can scroll over the page when the menu is open, which isn't a great user experience. If you would like to improve further this menu, I suggest you prevent the body from scrolling when it is open. You can achieve this by toggling a class that adds "overflow: hidden" to the body and removes it when hamburger/close buttons are clicked respectively or adding the style directly as inline. 🤓 🛠️ 

This is just an idea, feel free to implement your own solution! 🤓

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @danifromecuador, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @danifromecuador , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**